homeassistant/components/hyundai_kia_connect/entity.py

```diff
@@ -0,0 +1,28 @@
+"""Base Entity for Hyundai / Kia Connect integration."""
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import BRANDS, DOMAIN, REGIONS
+
+
+class HyundaiKiaConnectEntity(CoordinatorEntity):
+    """Class for base entity for Hyundai / Kia Connect integration."""
+
+    def __init__(self, coordinator, config_entry: ConfigEntry):
+        """Initialize."""
+        super().__init__(coordinator)
+        self.config_entry = config_entry
+
+    @property
+    def unique_id(self):
+        """Return a unique ID to use for this entity."""
+        return self.config_entry.entry_id
```

Comment:
- Don't we have a unique entry of the car?
- What if we have multiple entities of the same platform, how does this make the entity unique per entity?

Reply:
you are right about unique ID, fixed. regarding multiple entities, not in scope of initial integration.

Reply:
should be fixed now